Add docsite_pr and namespace_labels

dagwieers opened the pull request:
At some point it could be useful to add some additional logic, e.g. based on label colors or label descriptions. Or use a blacklist, rather than a whitelist (to protect labels that can cause privilege escalation).

jctanner commented on ansibullbot/triagers/plugins/label_commands.py:
```diff
@@ -7,24 +7,43 @@ def get_label_command_facts(issuewrapper, meta, module_indexer, core_team=[], va
         'openstack',
+        'ovirt',
+        'ucs',
+        'unarchive',
```
why is unarchive a namespace label?

dagwieers replied:
It's a "module community" label ? Much like the `xml` label.
It allows to track issues/PRs related to specific modules.

dagwieers replied:
I don't mind removing them from the list if that is an issue.

IMO a blacklist protecting special labels is more convenient than listing namespace labels (and others).

dagwieers replied:
Idea: maybe we could introduce m:<module> for module namespaces and only have ansibot add those when they were pre-created ? This way it's still on-demand, but integrates automatically with ansibot.

jctanner replied:
works for me, shipit.

dagwieers commented:
@jctanner I added support for module labels (`m:<module>`). So from now on people can add/remove those labels if they were pre-created. I also added `m:unachive` and `m:xml`.

dagwieers commented:
@jctanner If you want to shipit, merge it ;-)
